notebooks/failure-type-classification/metric_template.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>If we there is no option to call this notebook from its original location, can we customize it to be "helper for failure type" so it is its own notebook? And remove anything that's not needed for the workflow here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>